src/spyglass/position/position_merge.py

```diff
@@ -79,7 +79,13 @@ def fetch1_dataframe(self) -> DataFrame:
         return query.fetch1_dataframe()
 
     def fetch_pose_dataframe(self):
-        key = self.merge_restrict(self.proj()).proj()
+        """
+        Fetch a single dataframe of pose bodypart coordinates.
+        This will work only for sources that have pose information.
+        """
```

Comment:
The [dostring style guide](https://numpydoc.readthedocs.io/en/latest/format.html) spyglass follows recommends separating the oneliner from the rest.
```suggestion
        """Fetch a single dataframe of pose bodypart coordinates.
        
        This will work only for sources that have pose information.
        """
```